zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -126,6 +133,8 @@ pub(super) const TESTNET_ACTIVATION_HEIGHTS: &[(block::Height, NetworkUpgrade)]
     (block::Height(1_028_500), Canopy),
     (block::Height(1_842_420), Nu5),
     (block::Height(2_976_000), Nu6),
+    // FIXME: TODO: Set a correct value for NU7
+    (block::Height(2_942_001), Nu7),
```

Comment:
@arya2 ([Link](https://github.com/QED-it/zebra/pull/15#discussion_r1823376833)): It would be better to defer setting this at all until the release PR for a version that deploys NU7 on Testnet and to set this in the configuration until then.

`NetworkUpgrade::current()` will return NU6 as the current network upgrade if the activation height is higher than NU7, so this should be set above the NU6 activation height.


```suggestion
    // (block::Height(2_976_001), Nu7),
```

Reply:
@PaulLaux ([Link](https://github.com/QED-it/zebra/pull/15#discussion_r1824208010)): or consider putting this under the nu7 flag :`#[cfg(zcash_unstable = "nu6" /* TODO nu7 */ )]`

Reply:
@dmidem ([Link](https://github.com/QED-it/zebra/pull/15#discussion_r1912240988)): Link to this comment copied to #37